sf-stream/src/convert.rs

```diff
@@ -497,11 +498,24 @@ pub fn convert_ed25519_signature(sig: &Ed25519Signature) -> extractor::Ed25519Si
 pub fn convert_multi_ed25519_signature(
     sig: &MultiEd25519Signature,
 ) -> extractor::MultiEd25519Signature {
+    let public_key_indices: Vec<usize> = BitVec::from(sig.bitmap.0.clone()).iter_ones().collect();
     extractor::MultiEd25519Signature {
-        public_keys: sig.public_keys.iter().map(|pk| pk.0.clone()).collect(),
+        public_keys: public_key_indices
+            .iter()
+            .map(|index| {
+                sig.public_keys
+                    .get(*index)
+                    .unwrap_or_else(|| panic!("Failed to get public key at position {}", index))
+                    .0
+                    .clone()
+            })
+            .collect(),
```

Comment:
I'm not sure we want to change this one- otherwise what's the point of having the bitmap/public_key_indices?

Reply:
We should talk more about how to consume these but I can remove them for now.